drivers/dma/dma_mcux_edma.c

```diff
@@ -115,7 +114,8 @@ static void channel_irq(edma_handle_t *handle)
 				tcds_done = 0;
 			}
 		} else {
-			tcds_done = (uint32_t)new_header - (uint32_t)handle->header;
+			tcds_done =
+				(uint32_t)new_header - (uint32_t)handle->header;
```

Comment:
This whitespace change should also move. You can put all the whitespace changes together in one commit, just keep them separate from functional changes.

Reply:
revert back